lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -6,11 +6,11 @@ module DateHelper
       alias_method :_distance_of_time_in_words, :distance_of_time_in_words
       alias_method :_time_ago_in_words, :time_ago_in_words
 
-      include DOTIW::Methods
-
       def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options = {}, options = {})
         return _distance_of_time_in_words(from_time, to_time, options) if options.delete(:vague)
-        super
+        dotiw_methods = Object.new
```

Comment:
Well, that works but we don't want to be creating a new object just to call the method every time. Try a better solution, for example I think `ActionController::Base.helpers.extend DOTIW::Methods` or something similar might work. If you can't find the answer I'll spend some time on this.

Reply:
Fixed: https://github.com/radar/distance_of_time_in_words/pull/111/commits/0636e3d3740f2656a267a03c56f7ce2082363d11